pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -1,114 +1,167 @@
+        ++ lib.map (n: "FullyQualifiedName!=${n}") (
+          [
+            "NexusMods.UI.Tests.ImageCacheTests.Test_LoadAndCache_RemoteImage"
+            "NexusMods.UI.Tests.ImageCacheTests.Test_LoadAndCache_ImageStoredFile"
+
+            # TODO: investigate these test fails...
+            # Looks like the tests are still trying to use the bundled 7zz?
+            #
+            # Failed to start a process with file path '/build/source/tests/NexusMods.DataModel.Tests/bin/Release/net8.0/runtimes/linux-x64/native/7zz'.
+            # Target file or working directory doesn't exist, or the provided credentials are invalid.
+
+            # Failed with No Extractors found for file data_7zip_lzma2.7z
+            "NexusMods.CLI.Tests.VerbTests.ExtractArchive.CanExtractArchive"
+            "NexusMods.CLI.Tests.VerbTests.AnalyzeArchive.CanAnalyzeArchives"
+
+            # Failed with No Extractors found for file StubbedGameState.zip
+            "NexusMods.CLI.Tests.VerbTests.ModManagementVerbs.CanCreateAndManageLists"
+            "NexusMods.DataModel.Tests.ToolTests.CanRunTools"
+            "NexusMods.DataModel.Tests.FileHashCacheTests.UpdatingHashesReturnsTheUpdate"
+            "NexusMods.DataModel.Tests.FileHashCacheTests.CanGetHashOfSingleFile"
+            "NexusMods.DataModel.Tests.FileHashCacheTests.CanHashFolder"
+            "NexusMods.DataModel.Tests.FileOriginRegistryTests.RegisterFolder_WhenCalledTwice_ShouldOnlyAppendNewData"
+          ]
+          ++ lib.optionals (!enableUnfree) [
+            "NexusMods.Games.FOMOD.Tests.FomodXmlInstallerTests.InstallsFilesSimple_UsingRar"
+          ]
+        );
+    in
+    [
+      "--environment=USER=nobody"
+      "--filter=${lib.strings.concatStringsSep "&" filters}"
+    ];
 
   passthru = {
     tests = {
       serve = runCommand "${pname}-test-serve" { } ''
-        ${nexusmods-app}/bin/${nexusmods-app.meta.mainProgram}
+        ${nexusmods-app}/bin/${meta.mainProgram}
         touch $out
       '';
       help = runCommand "${pname}-test-help" { } ''
-        ${nexusmods-app}/bin/${nexusmods-app.meta.mainProgram} --help
+        ${nexusmods-app}/bin/${meta.mainProgram} --help
         touch $out
       '';
       associate-nxm = runCommand "${pname}-test-associate-nxm" { } ''
-        ${nexusmods-app}/bin/${nexusmods-app.meta.mainProgram} associate-nxm
+        ${nexusmods-app}/bin/${meta.mainProgram} associate-nxm
         touch $out
       '';
       list-tools = runCommand "${pname}-test-list-tools" { } ''
-        ${nexusmods-app}/bin/${nexusmods-app.meta.mainProgram} list-tools
+        ${nexusmods-app}/bin/${meta.mainProgram} list-tools
         touch $out
       '';
     };
     updateScript = ./update.bash;
   };
 
   meta = {
-    description = "Game mod installer, creator and manager";
+    description =
+      "Game mod manager, "
+      + (if enableUnfree then "includes" else "use the unfree version if you need")
+      + " support for mods packaged in RAR files";
```

Comment:
Please just keep the simple description to keep things simple.

Reply:
Ah, I wasn't aware of `longDescription`. Perhaps that is a more suitable place for this kind've information?